srv/components/system/files/confstore_template.j2

```diff
@@ -28,9 +28,9 @@ cluster>{{ cluster_id }}>storage_set[{{ outer_loop.index - 1 }}]>name=>{{ storag
 cluster>{{ cluster_id }}>storage_set[{{ outer_loop.index - 1 }}]>server_nodes[{{ loop.index - 1 }}]=>{{ pillar['cluster'][node]['machine_id'] }}
 cluster>{{ cluster_id }}>storage_set[{{ outer_loop.index - 1 }}]>storage_enclosures[{{ loop.index - 1 }}]=>{{ pillar['storage'][pillar['cluster'][node]['storage']['enclosure']]['enclosure_id'] }}
 {%- endfor %}
-cluster>{{ cluster_id }}>storage_set[{{ outer_loop.index - 1 }}]>durability>sns>data=>4
-cluster>{{ cluster_id }}>storage_set[{{ outer_loop.index - 1 }}]>durability>sns>parity=>2
-cluster>{{ cluster_id }}>storage_set[{{ outer_loop.index - 1 }}]>durability>sns>spare=>2
+cluster>{{ cluster_id }}>storage_set[{{ outer_loop.index - 1 }}]>durability>sns>data=>{{ pillar['cluster']['srvnode-1']['storage']['durability']['sns']['data'] }}
```

Comment:
I hope we pick correct values based on 1/3 node setup?

Reply:
Yes we pick values as user specify in our config.ini file.